pynecone/components/datadisplay/code.py

```diff
@@ -44,7 +44,7 @@ def _get_imports(self) -> imports.ImportDict:
         merged_imports = super()._get_imports()
         if self.theme is not None:
             merged_imports = imports.merge_imports(
-                merged_imports, {PRISM_STYLES_PATH: {self.theme.name}}
+                merged_imports, {PRISM_STYLES_PATH: {ImportVar(tag=self.theme.name)}}
```

Comment:
This seems to be causing issues when running pcweb but not sure why - seems `Prism` is being added as both a default and non-default import

```
./pages/docs/library/datadisplay/codeblock.js                                                                                                                                                       [44/1189]
Error:                                                                                                                                                                                                       
  x the name `Prism` is defined multiple times                                                                                                                                                               
    ,----                                                                                                                                                                                                    
 10 | import Prism, {Prism} from "react-syntax-highlighter"                                                                                                                                                  
    :        ^^|^^   ^^|^^                                                                                                                                                                                   
    :          |       `-- `Prism` redefined here                                                                                                                                                            
    :          `-- previous definition of `Prism` here                                                                                                                                                       
    `----       
```